client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -261,8 +262,11 @@ class Upload extends React.Component {
 			return this.renderNotAvailableForMultisite();
 		}
 
+		const title = translate( 'Themes > Install' );
+
 		return (
 			<Main>
+				<PageViewTracker path="/themes/upload/:site" title={ title } />
```

Comment:
The "title" here is used internally by the PageViewTracker component for composing the tracking event for GA and Tracks. Thus, it shouldn't be translated here, e.g. https://github.com/Automattic/wp-calypso/blob/master/client/me/help/main.jsx#L246 . This should suffice:

```
<PageViewTracker path="/themes/upload/:site" title="Themes > Install" />
```